packages/core/test/unit/instance-validator.test.js

```
@@ -158,4 +159,62 @@ describe(Support.getTestDialectTeaser('InstanceValidator'), () => {
+          },
+        },
+      });
+    });
+
+    it('accepts a valid JSON object string when isJSON is true', async function () {
+      const instance = JsonUser.build({ data: '{"key":"value"}' });
+      await expect(instance.validate()).not.to.be.rejectedWith(
+        Error,
+        /TypeError/,
+      );
+    });
```

ESLint

[error] Unexpected function expression.

(prefer-arrow-callback)